service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -970,6 +1026,15 @@ private OrdersDataForUserDto getOrdersData(Order order) {
 
         Double amountBeforePayment = convertCoinsIntoBills(amountWithDiscountInCoins - paidAmountInCoins);
 
+        double refundedBonuses = order.getPayment().stream()
+            .filter(payment -> ENROLLMENT_TO_THE_BONUS_ACCOUNT_ENG.equals(payment.getReceiptLink()))
+            .map(payment -> payment.getAmount().doubleValue())
+            .reduce(0.0, Double::sum);
+
+        refundedBonuses = refundedBonuses == 0.0 ? 0.0 : refundedBonuses / (-100);
+
+        Double refundedMoney = order.getRefund() == null ? 0.0 : -order.getRefund().getAmount().doubleValue() / 100;
```

Comment:
why do you divide by `-100` for `refundedBonuses` and by `100` for `refundedMoney`?
Please move -100 to the constant and use it for both expresions